css/components/navbar.css

```diff
@@ -0,0 +1,100 @@
+.nav__bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3.25rem;
+}
+
+.nav__logo__container {
+  margin: 0 1.25rem 0 0;
+}
+
+.nav__logo__img {
+  height: 3.25rem;
+  width: auto;
+}
+
+.nav__toggle__button span {
+  width: 3.25rem;
+  font-size: 2rem;
+  text-align: center;
+  padding: 0;
+  cursor: pointer;
+}
+
+@media screen and (min-width: 1024px) {
+  .nav__toggle__button {
+    display: none;
+  }
+}
+
+.nav__toggle__button:focus+.nav__menu,
+.nav__toggle__button:focus-within+.nav__menu {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: normal;
+  align-items: center;
+  background-color: #FFF;
+  padding: 1rem 1.25rem;
+  align-self: start;
+
+  .nav__list {
+    flex-direction: column;
+  }
+
+  .nav__button__group {
+    flex-direction: column;
+  }
```

Comment:
remember this is plain css, not sure if the container feature is working by default, but for now, let's not take advantage of it, these lines should be written as `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__list {` and `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__button__group {`

Reply:
> remember this is plain css, not sure if the container feature is working by default, but for now, let's not take advantage of it, these lines should be written as `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__list {` and `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__button__group {`

This works by default but I can separate this thanks for the feedback 🙇